### PR TITLE
Fixed wording AdminInvoicesController.php

### PR DESCRIPTION
**Pull request by Nobodaddy**
| Questions | Answers |
| --- | --- |
| Branch? | 1.7 |
| Description? | corrected some English strings |
| Type? | improvement |
| Category? | BO |
| BC breaks? | Does it break backward compatibility? yes/no |
| Deprecations? | Does it deprecate an existing feature? yes/no |
| Fixed ticket? | (optional) If this PR fixes a [Forge](http://forge.prestashop.com/) ticket, please add its complete Forge URL. |
| How to test? | Please indicate how to best verify that this PR is correct. |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

Improved English strings

### CODE REVIEW

**Comment by maximebiloe** [comment]
Thank you @Nobodaddy.

@AlexEven could you give us your opinion about the changes, please?

File: controllers/admin/AdminInvoicesController.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public function __construct()
                     ),
                     'PS_INVOICE_LEGAL_FREE_TEXT' => array(
                         'title' => $this->trans('Legal free text', array(), 'Admin.OrdersCustomers.Feature'),
-                        'desc' => $this->trans('Use this field to display additional text on your invoice, like specific legal information. It will appear below the payment methods summary.', array(), 'Admin.OrdersCustomers.Help'),
+                        'desc' => $this->trans('Use this field to show additional informations on the invoice,below the payment methods summary (like e.g. specific legal informations).', array(), 'Admin.OrdersCustomers.Help'),
```
Comment (AlexEven) [inline controllers/admin/AdminInvoicesController.php]: We're missing a space before "below".
Aren't "like" and "e.g." redundant?

Comment (Nobodaddy) [inline controllers/admin/AdminInvoicesController.php]: I agree. Added a space in the commit and and shortened to "like".

Comment (AlexEven) [inline controllers/admin/AdminInvoicesController.php]: Thank you Rainer :)
I hadn't noticed but "information" should be singular too. Could you please change it too? We'll merge after this!

Comment (Nobodaddy) [inline controllers/admin/AdminInvoicesController.php]: @AlexEven 
Done! :)

Comment (AlexEven) [inline controllers/admin/AdminInvoicesController.php]: @Nobodaddy Thank you! There are 2 occurrences, so there's still an extra "s" unfortunately ^^

**Comment by antoin-m** [comment]
Thanks :)

**Pull request merged**